app/lib/import_export.rb

```diff
@@ -153,16 +153,7 @@ def update_assessment_symptom_data(data, patients)
     data[:assessments][:checked].delete(:symptoms)
     data[:assessments][:headers].delete('Symptoms Reported')
 
-    # For large queries (e.g. 100k patients) of the `Symptom.where(...) query below
-    # we saw MySQL give up on queries and throw an exception.
-    # The workaround for this is to collect the symptom ids in a set and find them
-    # using batches of patients.
-    symptom_ids = Set.new
-    patients.pluck(:id).in_groups_of(10_000, false) do |batch|
-      symptom_ids << Symptom.where(condition_id: ReportedCondition.where(assessment_id: Assessment.where(patient_id: batch).pluck(:id)).pluck(:id)).pluck(:id)
-    end
-    symptom_names_and_labels = Symptom.where(id: symptom_ids.to_a.flatten).where.not(label: nil).where.not(name: nil)
-                                      .distinct.order(:label).pluck(:name, :label).transpose
+    symptom_names_and_labels = patients.joins(assessments: { reported_condition: :symptoms }).distinct.order(:label).pluck(:name, :label).transpose
```

Comment:
This query still needs to filter out symptom where the `name` OR `label` are `nil`

Reply:
@timwongj Good catch. Just added that back in.